staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -123,6 +122,42 @@ func MergeAPIResourceConfigs(
 		}
 	}
 
+	// Iterate through all group/version/resource overrides specified in runtimeConfig.
```

Comment:
While this looks good now, I still wonder whether we can and should make this optional, opt-in for each apiserver.

Reply:
Or even move this into the kube-apiserver variant.

Reply:
Compare https://github.com/kubernetes/kubernetes/pull/62254/files#r180091149

Reply:
> Or even move this into the kube-apiserver variant.

Yes.  This is a kube-apiserver variant.  Removing individual resources is likely to lead to sadness since users don't usually understand how individual pieces depend on each other.

Reply:
ok, make this function(enable/disable resource) only work for kube-apiserver.